interface/codeview.py

```diff
@@ -0,0 +1,26 @@
+import io
+import logging
+from zipfile import ZipFile
+
+from interface.backend.minio_api import MissingFile, download_buffer
+
+log = logging.getLogger(__name__)
+
+
+def submission_cut(request, submission, filename):
```

Comment:
Q: From what comes the name ```submission_cut```?

Reply:
It was the preliminary form of the function. It's actually `extract_file`.